src/doc/partials/type.hbs

```diff
@@ -83,9 +87,12 @@
 <h3>Title</h3>
 <div class="win-type-title">...</div>
 
-<h4>Subtitle</h4>
+<p class="win-type-subtitle">Subtitle</p>
```

Comment:
This particular class is a case of backwards compatibility. win-type-subtitle is 20px and is going to be used less, win-type-subtitle-alt is 18px is going to be used more and is designed to give more contrast in size when compared to win-type-title. This is why win-type-subtitle is not mapped to <h4> and win-type-subtitle-alt is.